scripts/bump_version.sh

```diff
@@ -81,6 +81,10 @@ then
   exit 1
 fi
 
+[ $VERBOSE ] && echo "Bumping VERSION in CMakeLists.txt"
+sed -e "s/ VERSION [0-9.]\{1,\}/ VERSION $VERSION/g" < CMakeLists.txt > tmp
```

Comment:
Why not just use `sed -i`?

Reply:
Ah, I see that it's done like this all over the file. Is this just for consistency?